README.md

```diff
@@ -1,25 +1,42 @@
-Wagn application code used at Wikirate.org
+Decko application code used at Wikirate.org
 =========
 
 Steps to make it work
 ----
 
-1. create folders "files","config","public" and "public/assets" under the wikirate root directory.
-2. rake wagn:seed
-3. import the database
-4. bundle exec wagn s
+The following will help set up a functioning wikirate site with a small subset of (mostly fake) data.  Some pages will not look complete.
+
+1. clone repo: `git clone git@github.com:wikirate/wikirate.git`
+1. enter dir: `cd wikirate`
+1. init/update submodules `git submodule update -f --init --recursive`
+1. install gems: `bundle install`
+1. set up config: `cp -R sample_config/* config`
+1. populate dev database (with test data): `DATABASE_NAME_TEST=wikirate_dev bundle exec rake wikirate:test:seed`
+1. add AWS credentials to config/application.rb (Ask wikirate dev team!  Sorry, we'll make this easier soon)
```

Comment:
I'm not in love with this step, but it should be fine for Monday.

I'm thinking I should update our credentials and give each of us our own.  Also, I think we should probably make it so that none of our dev credentials empower us to alter anything in the live bucket.

Reply:
I use `development: *test` in my database.yml instead of seeding the dev db with test data.

Reply:
I'm using the user "reader" in my dev credentials. I don't think I can alter anything.